docs/products/clickhouse/howto/sharding.rst

```diff
@@ -0,0 +1,74 @@
+Work with shards in ClickHouse®
+=================================
+
+Learn why and how use shards and find out how shards get allocated to Aiven services.
+
+About sharding
+--------------
+
+Sharding is a mechanism for splitting a database into shards across multiple servers. A shard is a horizontal partition of data consisting of one or more replicas. Each replica of the shard can handle queries sent to this shard since there's no master. Sharding enables horizontal cluster scaling, which helps spread out the load, increase processing speed, and support more traffic.
+
+Enable shards
+-------------
+
+With Aiven services, shards are created automatically when the service is created. The number of shards that your service gets depends on the plan you select for your service. You can calculate the shards number as follows:
+
+.. math::
+
+  number\_of\_shards = number\_of\_nodes / 3
+
+Use shards
+----------
+
+**Prerequisites**
+
+Create cluster *test-cluster* with 5 shards and database *test-db*.
+
+Create a distributed table
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. topic:: Shards vs. distributes tables
```

Comment:
We haven't used "topic" anywhere else.  What is information goal of this section? It doesn't seem to fit into "howto", are we talking about concept?